buildout.cfg

```diff
@@ -95,11 +106,9 @@ eggs = ${instance:eggs}
 
 
 [test]
-recipe = zc.recipe.testrunner
+recipe = collective.xmltestreport
+defaults = ['--auto-color', '--auto-progress', '--ignore_dir=.git', '--ignore_dir=bower_components', '--ignore_dir=node_modules']
```

Comment:
`'-s', 'plonetheme.tokyo',` is missing -> sidebar tests are running -> was this intentional or by accident?